app/models/concerns/assign_to_current_semester.rb

```diff
@@ -6,6 +6,7 @@ module AssignToCurrentSemester
   end
 
   def assign_to_current_semester
-    self.semester = Semester.current_semester unless semester.present?
+    # TODO: Settings are not initilized yet here
+    # self.semester = Semester.find(Settings.new.current_semester_id)
```

Comment:
i didn't end up actually using the config/initializers. maybe i should look more into that. feels bad to have a dummy semester tho